README.md

```diff
@@ -32,6 +32,14 @@ console.log(one) // [1]
 
 function swap(array, first_index, second_index) {
   // -- YOUR ANSWER HERE --
+    if (array.length >2){
+  var temp = array[first_index];
+  array[first_index] = array[second_index];
+  array[second_index] = temp
+  return array
+  }else{
+    return array
+  }
```

Comment:
Since the fn returns the array either way, I would've put the return outside of the conditional.